node.js

```diff
@@ -372,7 +372,8 @@ module.exports = {
     if (latest !== 0 && latest < halfYearAgo) {
       console.warn(
         'Browserslist: caniuse-lite is outdated. Please run:\n' +
-        'npx browserslist@latest --update-db'
+        'npx browserslist@latest --update-db\n\n' +
+        'See https://github.com/browserslist/browserslist#why-you-need-to-call-update-db-script-regularly for details'
```

Comment:
The URL is too long. It will be split into 2 lines and will into work in console.

Reply:
Excuse my ignorance but is there a way I can keep the title but have a shorter anchor for use in the markdown and URL?

Or do you recommend a different title for the section?

Reply:
I think we can do few steps:

1. Change text to `See for details:\nLINK`
2. Try to make the title a little shorter
3. Double-check that URL length is <= 76 symbols (80 minus 4 symbols indent for some environments)